blob/drivertest/drivertest.go

```diff
@@ -1830,9 +1831,6 @@ func testAs(t *testing.T, newHarness HarnessMaker, st AsTest) {
 	if gotErr == nil {
 		t.Fatalf("got nil error from NewReader for nonexistent key, want an error")
 	}
-	if err := st.ErrorCheck(gotErr); err != nil {
```

Comment:
This should not have been deleted.

Reply:
Oops, just saw this. will fix.

Reply:
#1167

Reply:
Sorry, I tend to hover above the "Squash and Merge" button ready to spring into action when it goes green.